cms/static/js/views/pages/container.js

```diff
@@ -868,12 +968,32 @@ function($, _, Backbone, gettext, BasePage,
                     || (useNewProblemEditor === 'True' && blockType.includes('problem')))
             ){
                 var destinationUrl;
-                if (useVideoGalleryFlow === "True" && blockType.includes("video")) {
-                    destinationUrl = this.$('.xblock-header-primary').attr("authoring_MFE_base_url") + '/course-videos/' + encodeURI(data.locator);
+                var pathToXBlockEditor;
+                if (useVideoGalleryFlow === 'True' && blockType.includes('video')) {
+                    pathToXBlockEditor = `/course-videos/${encodeURI(data.locator)}`;
+                    destinationUrl = `${this.$('.xblock-header-primary').attr('authoring_MFE_base_url')}${pathToXBlockEditor}`;
                 }
                 else {
-                    destinationUrl = this.$('.xblock-header-primary').attr("authoring_MFE_base_url") + '/' + blockType[1] + '/' + encodeURI(data.locator);
+                    pathToXBlockEditor = `/${blockType[1]}/${encodeURI(data.locator)}`;
+                    destinationUrl = `${this.$('.xblock-header-primary').attr('authoring_MFE_base_url')}${pathToXBlockEditor}`;
                 }
+
+                try {
+                    if (this.options.isIframeEmbed) {
+                        return window.parent.postMessage(
+                            {
+                                type: 'newXBlockEditor',
+                                message: 'Sends a message when the new XBlock editor is opened',
+                                payload: {
+                                    url: pathToXBlockEditor,
```

Comment:
Again, I don't think we should be passing URLs in events. Can we just pass the block type and usageId, and let the MFE decide if it's going to be `/course-videos/...` or `/video/...` ?

Reply:
Deleted sending a post message, since this method is not used.
[[inform] Implemented in follow-up commit: https://github.com/openedx/edx-platform/commit/8d5b7e3bb77cf48352f1e7864636ac05b31810e1#diff-1b7855448b6086c2a313e0b9ecff179e991106aac52648d880f6f9d551e64577R680](https://github.com/openedx/edx-platform/pull/35694/commits/8d5b7e3bb77cf48352f1e7864636ac05b31810e1#diff-1b7855448b6086c2a313e0b9ecff179e991106aac52648d880f6f9d551e64577L977)